src/components/Tinymce/index.vue

```diff
@@ -100,7 +100,8 @@ export default class extends Vue {
     es: 'es',
     ja: 'ja',
     ko: 'ko_KR',
-    it: 'it'
+    it: 'it',
+    br: 'br'
```

Comment:
please copy pt_BR file in https://www.tiny.cloud/get-tiny/language-packages/ to the public folder as mentioned above, also change this to pt_BR